zkevm-circuits/src/evm_circuit/util/common_gadget/tx_l1_fee.rs

```diff
@@ -407,6 +448,8 @@ mod tests {
                 is_curie,
                 gadget,
                 tx_data_gas_cost,
+                // #[cfg(feature = "l1_fee_curie")]
```

Comment:
#[cfg(feature = "l1_fee_curie")] can be removed

Reply:
removed.